core/src/test/java/kafka/test/ClusterInstance.java

```diff
@@ -160,15 +159,11 @@ default Admin createAdminClient() {
     }
 
     default Set<GroupProtocol> supportedGroupProtocols() {
-        Map<String, String> serverProperties = config().serverProperties();
-        Set<GroupProtocol> supportedGroupProtocols = new HashSet<>();
-        supportedGroupProtocols.add(CLASSIC);
-
-        if (serverProperties.getOrDefault(GROUP_COORDINATOR_REBALANCE_PROTOCOLS_CONFIG, "").contains("consumer")) {
-            supportedGroupProtocols.add(CONSUMER);
+        if (isKRaftTest()) {
```

Comment:
The `CONSUMER` protocol can be disabled even though it is kraft mode, right? Maybe we can check the `KafkaAPIs` directly. for example:
```java
    default Set<GroupProtocol> supportedGroupProtocols() {
        if (brokers().values().stream().allMatch(b -> b.dataPlaneRequestProcessor().isConsumerGroupProtocolEnabled())) {
            return mkSet(CLASSIC, CONSUMER);
        } else {
            return Collections.singleton(CLASSIC);
        }
    }
```